examples/data-sources/ome_fbc_repository/datasource.tf

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://mozilla.org/MPL/2.0/
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Get details of all the firmware repositories 
+data "ome_fbc_repository" "fbc-repository-all" {
+
+}
+output "fbc-repository" {
+  value = data.ome_fbc_repository.fbc-repository-all
+}
+
+# data "ome_fbc_repository" "fbc-repository-name-filter" {
+#         names = [
```

Comment:
Could you add a comment above the names filter to give some details about what happens if filter is empty or invalid.